packages/pkp-base/src/lib/pkp-base.ts

```diff
@@ -324,11 +321,12 @@ export class PKPBase<T = PKPBaseDefaultParams> {
       throw new Error('pkpPubKey (aka. uncompressPubKey) is required');
     }
 
-    if (this.controllerAuthSig && this.controllerSessionSigs) {
-      throw new Error(
-        'controllerAuthSig, controllerSessionSigs and controllerAuthMethod are defined, can only use one authorization type'
+    this.validateAuthContext();
+
```

Comment:
shouldn't we have this code as well here?
```js
if (
  this.authContext &&
  typeof this.authContext?.client !== 'object' &&
  typeof this.authContext?.getSessionSigsProps !== 'function' &&
  !Array.isArray(this.authContext?.authMethods)
) {
  throw new Error('controllerSessionSigs must be an object');
}
```

Reply:
You are right. Added